src/logic/calculate.js

```diff
@@ -0,0 +1,16 @@
+import operate from 'operate';
```

Comment:
`./` add this, throwing error because of this.
imports from the same folder should starts with './'

Reply:
really? you could have approved the project and ask for that insignificant change...